katdal/test/test_datasources.py

```diff
@@ -296,3 +297,15 @@ def test_construction_from_url(self):
             open_data_source('ftp://unsupported')
         with assert_raises(DataSourceNotFound):
             open_data_source(rdb_filename[:-4])
+        source_name = f'{cbid}_{sn}'
+        assert source_from_file.name == source_name
+        assert rdb_filename in source_from_file.url
+
+    def test_name_change(self):
+        # Ideally this test should do the following
+        # >>> d = katdal.open(RDBfile)
```

Comment:
This kind of test will have to reside in `test_dataset` or `test_visdatav4`, not `test_datasources`.